set.mm

```diff
@@ -79592,21 +79592,544 @@ currently used conventions for such cases (see ~ cbvmpox , ~ ovmpox and
+    nffrecs.1 $e |- F/_ x R $.
+    nffrecs.2 $e |- F/_ x A $.
+    nffrecs.3 $e |- F/_ x F $.
+    $( Bound-variable hypothesis builder for the well-founded recursion
+       generator.  (Contributed by Scott Fenton, 23-Dec-2021.) $)
+    nffrecs $p |- F/_ x frecs ( R , A , F ) $=
+      ( vf vy vz cfrecs cv wfn wss cpred wral wa nfcv nfss nfralw cfv cres wceq
+      co w3a wex cab cuni df-frecs nfpred nfan nfres nfov nfeq2 nf3an nfex nfab
+      nfv nfuni nfcxfr ) ABCDKHLZILZMZVBBNZBCJLZOZVBNZJVBPZQZVEVAUAZVEVAVFUBZDU
+      DZUCZJVBPZUEZIUFZHUGZUHIJBCHDUIAVQVPAHVOAIVCVIVNAVCAURVDVHAAVBBAVBRZFSVGA
+      JVBVRAVFVBABCVEEFAVERZUJZVRSTUKVMAJVBVRAVJVLAVEVKDVSGAVAVFAVARVTULUMUNTUO
+      UPUQUSUT $.
+  $}
+
+  ${
+    $d A w y z $.  $d F w y z $.  $d G w y z $.  $d H w y z $.  $d R w y z $.
+    $( Functions defined by well-founded recursion over a partial order are
+       identical up to relation, domain, and characteristic function.  This
+       version of ~ frr3g does not require infinity.  (Contributed by Scott
+       Fenton, 24-Aug-2022.) $)
+    fpr3g $p |- ( ( ( R Fr A /\ R Po A /\ R Se A ) /\
+                     ( F Fn A /\
+                       A. y e. A ( F ` y ) =
+                                 ( y H ( F |` Pred ( R , A , y ) ) ) ) /\
+                     ( G Fn A /\
+                       A. y e. A ( G ` y ) =
+                                 ( y H ( G |` Pred ( R , A , y ) ) ) ) )
+                   -> F = G ) $=
+      ( vz vw w3a wfn cv cfv cres co wceq wral wa wi fveq2 eqeq12d wfr wpo wcel
+      wse cpred eqidd r19.21v simprll simprrl wss predss fvreseq mpan2 biimp3ar
+      wb syl2anc oveq2d predeq3 reseq2d oveq12d simp2lr rspcdva simp2rr 3eqtr4d
+      weq id simp1 3exp a2d syl5bi imbi2d frpoins2g sylib 3impib simp2l eqfnfv2
+      simp3l mpbir2and ) BCUABCUBBCUDIZDBJZAKZDLZWADBCWAUEZMZFNZOZABPZQZEBJZWAE
+      LZWAEWCMZFNZOZABPZQZIZDEOZBBOZGKZDLZWSELZOZGBPZWPBUFVSWHWOXCVSWHWOQZXBRZG
+      BPXDXCRXEXDHKZDLZXFELZOZRZGHBCXJHBCWSUEZPXDXIHXKPZRWSBUCZXEXDXIHXKUGXMXDX
+      LXBXMXDXLXBXMXDXLIZWSDXKMZFNZWSEXKMZFNZWTXAXNXOXQWSFXMXDXOXQOZXLXMXDQVTWI
+      XSXLUOZXMVTWGWOUHXMWHWIWNUIVTWIQXKBUJXTBCWSUKHBXKDEULUMUPUNUQXNWFWTXPOABW
+      SAGVEZWBWTWEXPWAWSDSYAWAWSWDXOFYAVFZYAWCXKDBCWAWSURZUSUTTVTWGWOXMXLVAXMXD
+      XLVGZVBXNWMXAXROABWSYAWJXAWLXRWAWSESYAWAWSWKXQFYBYAWCXKEYCUSUTTWIWNWHXMXL
+      VCYDVBVDVHVIVJGHVEZXBXIXDYEWTXGXAXHWSXFDSWSXFESTVKVLXDXBGBUGVMVNWPVTWIWQW
+      RXCQUOVSVTWGWOVOVSWHWIWNVQGBBDEVPUPVR $.
+  $}
+
+  ${
+    $d A f g w x y z $.  $d f G g w x y z $.  $d R f g w x y z $.
+    frrlem1.1 $e |- B = { f |
+                           E. x ( f Fn x /\
+                                  ( x C_ A /\
+                                    A. y e. x Pred ( R , A , y ) C_ x ) /\
+                                  A. y e. x ( f ` y ) =
+                                            ( y G ( f |` Pred ( R , A , y ) ) )
+                                 ) } $.
```

Comment:
This (and previous and next statements) can use fewer lines while retaining sensible line-breaks.

Reply:
Done